src/sentry/issues/escalating.py

```diff
@@ -41,6 +41,7 @@
 BUCKETS_PER_GROUP = 7 * 24
 ONE_WEEK_DURATION = 7
 IS_ESCALATING_REFERRER = "sentry.issues.escalating.is_escalating"
+GROUP_DAILY_COUNT_DURATION = 60
```

Comment:
```suggestion
GROUP_DAILY_COUNT_TTL = 60
```

A personal preference: TTL helps me know that it is for the purpose of caching. Up to you if you want to take it.